views/js/test/actionBar/button/test.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <title>Ui Test - Test Runner - Action Bar Button</title>
+        <base href="../../../../../../../../tao/views/" />
+        <link rel="stylesheet" type="text/css" href="js/lib/qunit/qunit.css">
+        <script type="text/javascript" src="js/lib/require.js"></script>
+        <script type="text/javascript" src="js/lib/qunit/qunit.js"></script>
+        <script type="text/javascript" src="js/lib/qunit/qunit-parameterize.js"></script>
+        <script type="text/javascript" src="js/lib/blanket/blanket.min.js" data-cover-only="actionBar/button.js"></script>
+
+        <script  type="text/javascript">
+            var taobase_www = '../../../../../../../../../tao/views/';
+            var base_lang = 'en';
```

Comment:
I'll remove them on a separate PR because it affects all the unit tests of the extension